packages/core/lib/console.js

```diff
@@ -143,16 +126,53 @@ class Console extends EventEmitter {
     abstractions.forEach(abstraction => {
       contextVars[abstraction.contract_name] = abstraction;
     });
+  }
+
+  runSpawn(inputStrings, options, callback) {
+    let args;
```

Comment:
`args` is confusing in this context because It implies some none zero number of arguments when, in this case, it is the module/entryPoint to spawn

Reply:
Agree. Renamed to `childPath` to be more clear.